### PR TITLE
Extend indexing with partial indexing

### PR DESCRIPTION
**Pull request by Beacze**
Depends on #1925

### CODE REVIEW

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 1]
```diff
@@ -699,6 +699,8 @@ search=Suche
 start=Start
 startDate=Startdatum
 startIndexing=Indexierung starten
+startIndexingAll=Indexierung starten f\u00FCr allen
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_de.properties]: Maybe better translation would be `Indexierung \u00FCber alles starten`

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 2]
```diff
@@ -699,6 +699,8 @@ search=Suche
...
+startIndexingRemaining=Indexierung starten f\u00FCr nicht indexiert
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_de.properties]: A maybe better translation would be `Indexierung f\u00FCr nicht Indexierte starten`

File: Kitodo/src/main/resources/messages/messages_de.properties (hunk 1 above)
Comment (Erikmitk) [reply Kitodo/src/main/resources/messages/messages_de.properties]: What about: "Komplette Indexierung starten"

File: Kitodo/src/main/resources/messages/messages_de.properties (hunk 2 above)
Comment (Erikmitk) [reply Kitodo/src/main/resources/messages/messages_de.properties]: Hmm… tough one… maybe `Indexierung vervollständigen`?

**Comment by Beacze** [comment]
I have changed translations and rebased on master.

**Pull request merged**